### ISSUE TITLE
[Bug]: Continuous Reconnect after deleting source

### ISSUE THREAD

**Issue by bdocrobot** [josephdadams/TallyArbiter#223]
### What happened?

I was doing some testing, and I added an instance of a BMD Atem, and I was monitoring the logs on a raspberry pi.  Since I didn't have a device for it to connect to at the moment, it started a reconnecting loop which is what I expected.  However, when I deleted the BMD Atem instance, the loop continues to run.  That I didn't expect.

### Version

2.0.14

### Distribution

CLI

### OS

Linux

### What browsers are you seeing the problem on?

Chrome

### TallyArbiter configuration

```javascript
not sure where this is
```

### Relevant log output

```shell
10/1/21 3:47 PM] The config file could not be found.
[10/1/21 3:47 PM] Tally Arbiter running on port 4455
[10/1/21 3:49 PM] Source Added: Atem
[10/1/21 3:50 PM] Device Added: Camera 1
[10/1/21 3:53 PM] Listener Client Connected. Type: web Device: Camera 1
[10/1/21 3:54 PM] Source Deleted: Atem
[10/1/21 4:11 PM] Source Added: sw 1
[10/1/21 4:11 PM] Source: sw 1 TSL 3.1 Server started. Listening for data on UDP Port: 4147
[10/1/21 4:11 PM] Device Source Added: Camera 1 - sw 1
[10/1/21 4:15 PM] Source: sw 1 TSL 3.1 UDP Server Stopped. Connection Closed.
[10/1/21 4:15 PM] Source Deleted: sw 1
[10/1/21 4:28 PM] Listener Client Connected. Type: web Device: Camera 1
```

**Comment by josephdadams** [comment]
Pretty sure we fixed this in 3.0 which is not yet released

**Comment by Jocke4f** [comment]
Is this still an issue on 3.0.5?

**Comment by Jocke4f** [comment]
I haven't seen this when tesing v3.0.7 with ATEM Mini Extreme ISO or ATEM Television Studio

My conclusion is that this was fixed in the code base for 3.0 and that we can close this.
